templates/donor-donation-success.html.twig

```diff
@@ -5,11 +5,11 @@
   <p>Thank you for supporting {{ charityName }} with your generous gift through the Big Give.<p>
   <p>Here is your donation information:</p>
   <ul>
-    <li>Total for {{ charityName }}: <strong>£{{ totalCharityValueAmount }}</strong></li>
-    <li>Donation: <strong>£{{ donationAmount}}</strong></li>
-    <li>Gift Aid: <strong>£{{ giftAidAmountClaimed }}</strong></li>
+    <li>Total for {{ charityName }}: <strong>{{ totalCharityValueAmount | format_currency('GBP', {min_fraction_digit: 0, max_fraction_digit: 2}) }}</strong></li>
```

Comment:
@NoelLH I guess we'd want to use this `format_currency` in the one area on pledge email template too

Reply:
Yep @tbg-jasonfung. I couldn't do both until one was reviewed while keeping the branches independent.

Reply:
No probs, just created a PR for it